Stripe3DS2/Stripe3DS2/Resources/Stripe3DS2.xcassets/discover-logo.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "discover@1x.png",
+      "idiom" : "universal"
+    }
```

Comment:
Why is it that some imagesets have only a single @1x and others have @1x, @2x, and @3x ?

Reply:
We don't have a 3x source asset for this image. When we do have one, we add all 3 (1x 2x and 3x).